src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/returncount/Example1.java

```diff
@@ -0,0 +1,74 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="ReturnCount">
+      <property name="max" value="3"/>
+    </module>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.coding.returncount;
+
+// xdoc section -- start
+import java.util.function.Predicate;
+
+public class Example1 {
+    public Example1() {
+    }
+
+    public Example1(int i) {
+        return;
+    }
+
+    String stringA = "String A";
+    String stringB = "String B";
+
+    public int signA() {
+        if (stringA.equals(stringB)) {
+            return -1;
+        }
+        return 0;
+    } // OK
+
+    public int signB(int x) {
+        if (stringA.equals(stringB)) {
+            return -1;
+        }
+        if (x == 0) {
+            return 0;
+        }
+        return 1;
+    } // OK
+
+    public int signC(int x) {
+        if (stringA.equals(stringB)) {
+            return -1;
+        }
+        if (x == 0) {
+            return 0;
+        }
+        if (x > 2) {
+            return 2;
+        }
+        return 1;
+    } // violation, more than three return statements
+
+    final Predicate<Integer> lambdaA = i -> {
+        if (i > 5) {
+            return true;
+        }
+        return false;
+    }; // OK
+
+    final Predicate<Integer> lambdaB = i -> {
+        return i > 5;
+    }; // OK
+
+    public void methodA(int x) {
+    }
+
+    public void methodB(int x) {
+        return;
+    }
+}
+// xdoc section -- end
```

Comment:
Please add new line at the end of all files.  To avoid this red signal

Reply:
Done